modules/user-guide/pages/using-vale-in-the-ide.adoc

```diff
@@ -24,13 +24,6 @@ Consider using Vale in your IDE when:
 
 . In  Visual Studio Code, install the link:https://marketplace.visualstudio.com/items?itemName=errata-ai.vale-server[Visual Studio Code extension for Vale using the Marketplace].
```

Comment:
Suggestions to simplify the first statement:

Open the VS Code Marketplace and install the link:https://marketplace.visualstudio.com/items?itemName=errata-ai.vale-server[Vale errata-ai extension].

Also spell out the abbreviation earlier.